lib/hexpm/accounts/backupcode.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Hexpm.Accounts.BackupCode do
+  @app_secret Application.get_env(:hexpm, :totp_encryption_secret)
```

Comment:
Avoid putting config in module attributes unless it's necessary. It's because changing config requires recompilation when in attributes.